### ISSUE TITLE
Incorrect Camera Connector Pin Count in Documentation?

### ISSUE THREAD

**Issue by Cosmo** [FrameworkComputer/Framework-Laptop-13#44]
In the [Mainboard drawings](https://github.com/FrameworkComputer/Framework-Laptop-13/blob/main/Mainboard/2D/fw_main_pcb_generic_2_w_fan.pdf) for the Framework Laptop 13, the Camera Connector is listed as 20879-040E-01 from I-PEX, which is a 40-pin connector. However, this seems to be incorrect, as the connector should be a 30-pin version.

The correct part number appears to be I-PEX 20879-030E-01. 
Could you please confirm if this is an error in the documentation or provide clarification?

**Comment by JohnAZoidberg** [comment]
You're correct. I think it's a labeling issue in the mechanical drawing, the name of the corrector is incorrect but it does say "30P".
You can find the correct part number here: https://github.com/FrameworkComputer/Framework-Laptop-13/tree/main/Mainboard#webcam-interface
